### ISSUE TITLE
Propose ECG telemetry tensor maps

### ISSUE THREAD

**Issue by erikr** [aguirre-lab/ml4c3#1233]
## What and why
Features from ECG telemetry might be interesting to use as predictors for code blue and rapid response events, intubation, etc.
This is also a good baseline prior to performing deep learning.

## Solution(s)
Propose initial TensorMaps that return features from ECG telemetry waveforms.

## Acceptance criteria
We have a list of features to extract, enabling a discussion with PIs.

## Blocked by or pending

**Comment by erikr** [comment]
@raimonpv We had two redundant issues that also were old and the history was cluttered, so I remade this.

At the last standup, @RidwanAlam said he would own this. I think a good first task is to propose which TensorMaps we want to build. Please add your thoughts too.

**Comment by raimonpv** [comment]
from what I remember, extracting the peaks and onsets for the whole wave was taking too much time to do it dynamically when training. Instead what I did was to create a `recipie` mode which did the computations and then stored the results in the hd5.
Nevertheless, the speed problem was mainly due to I was computing features for the whole signal. So I propose the following:
If we already now which chunks of ECG we are interested on or we now that we are going to work with small samples, we could create a tmap that computes dynamically the different features. If that's not the case I think that will be better to compute the features and store them in the hd5 file.

**Comment by erikr** [comment]
Good point. What length of ECG telemetry do you and @RidwanAlam envision needing?

If we do a sliding window approach where each sample consists of 10 sec of data, computing features is reasonably fast per window.

However, doing this on hundreds of windows will be slow.

**Comment by erikr** [comment]
Aaron would like @StevenSong @raimonpv and I to make progress training a convolutional neural network on ECG telemetry to predict arrests.

I will take over this issue.

The scope is now to develop a TMap `ecg_telemetry_<duration>_sec` that returns a numpy array of that many seconds of data.

The sampling frequency is important; we do not want this TMap to return twice as much data for a tele recording with 2x the sampling frequency.
